qa/tasks/thrashosds-health.yaml

```diff
@@ -28,3 +28,5 @@ overrides:
       - mons down
       - mon down
       - out of quorum
+      - noscrub
+      - nodeep-scrub
```

Comment:
I would check what this red mark means, but otherwise LGTM

Reply:
I think I need to add one more spacing